src/Console/InfectionContainer.php

```diff
@@ -196,6 +197,10 @@ public function __construct(array $values = [])
         $this['memory.formatter'] = function (): MemoryFormatter {
             return new MemoryFormatter();
         };
+
+        $this['infection.config.validator'] = function (): Validator {
+            return new Validator();
```

Comment:
Why can't it be initialized right here? E.g. just like it's done above for other dependencies with `$this->getInfectionConfig()`.

Reply:
Oh, nevermind. It'll be more difficult to pull out.